sherpa/astro/xspec/__init__.py

```diff
@@ -1359,23 +1359,26 @@ def __init__(self, filename, name='xstbl', parnames=(),
         self.addmodel = addmodel
         self.etable = etable
 
-        # Order appears to be
-        #   - z
+        # Order should be (z and escale were incorrectly mixed up
+        # in XSPEC 12.13.1 and earlier):
+        #
         #   - Escale
+        #   - z
         #   - norm
+        #
         # (for those models that support the relevant value)
         #
-        if addredshift:
-            self.redshift = XSBaseParameter(name, 'redshift', 0., 0., 5.,
-                                            0.0, 5, frozen=True)
-            pars.append(self.redshift)
-
         if addescale:
             # Should this just use Parameter?
             self.Escale = XSParameter(name, 'Escale', 1, 0, 1e20,
                                       0, 1e24, frozen=True, units="keV")
```

Comment:
Actually, it should be

```suggestion
            self.Escale = XSParameter(name, 'Escale', 1, 1e-5, 100,
                                      1e-5, 100, frozen=True)
```